src/layouts/hass-tabs-subpage-data-table.ts

```diff
@@ -78,12 +87,18 @@ export class HaTabsSubpageDataTable extends LitElement {
    */
   @property({ type: String }) public filter = "";
 
+  @property() public searchLabel?: string;
+
   /**
    * List of strings that show what the data is currently filtered by.
    * @type {Array}
    */
   @property({ type: Array }) public activeFilters?;
 
+  @property() public hiddenLabel?: string;
```

Comment:
Can you add a description to this property. Not clear from the name what it's for.